Sources/Public/DynamicProperties/AnimationKeypath.swift

```diff
@@ -44,6 +46,14 @@ public struct AnimationKeypath: Hashable, ExpressibleByStringLiteral {
     self.keys = keys
   }
 
-  var keys: [String]
+  // MARK: Public
+
+  /// The dot-separated key values that represent this keypath.
+  public internal(set) var keys: [String]
```

Comment:
Added some more public helpers to this type since it has a bigger role in the public API of `AnimationKeypathTextProvider` now.